src/components/ContentUploader/progressCellRenderer.js

```diff
@@ -18,12 +18,19 @@ type Props = {
  * Get error message for a specific error code
  *
  * @param {string} [errorCode]
+ * @param {string} [itemName]
  * @returns {FormattedMessage}
  */
-const getErrorMessage = (errorCode: ?string) => {
+const getErrorMessage = (errorCode: ?string, itemName: ?string) => {
     switch (errorCode) {
+        case ERROR_CODE_CHILD_FOLDER_FAILED_UPLOAD:
+            return <FormattedMessage {...messages.uploadsOneOrMoreChildFoldersFailedToUploadMessage} />;
         case 'file_size_limit_exceeded':
             return <FormattedMessage {...messages.uploadsFileSizeLimitExceededErrorMessage} />;
+        case 'item_name_invalid':
```

Comment:
Can you change these or file a bug to change these to constants too.

Reply:
Will move to constants in this PR.